packages/middleware-flexible-checksums/src/validateChecksumFromResponse.ts

```diff
@@ -31,7 +33,17 @@ export const validateChecksumFromResponse = async (
     const responseHeader = getChecksumLocationName(algorithm);
     const checksumFromResponse = responseHeaders[responseHeader];
     if (checksumFromResponse) {
-      const checksumAlgorithmFn = selectChecksumAlgorithmFunction(algorithm as ChecksumAlgorithm, config);
+      let checksumAlgorithmFn: ChecksumConstructor | HashConstructor;
+      try {
+        checksumAlgorithmFn = selectChecksumAlgorithmFunction(algorithm as ChecksumAlgorithm, config);
+      } catch (error) {
+        if (algorithm === ChecksumAlgorithm.CRC64NVME) {
+          logger?.warn(`Skipping ${ChecksumAlgorithm.CRC64NVME} checksum validation: ${error.message}`);
```

Comment:
currently, the only possible way to enter this branch is due to the thrown missing dependency error, but should this perhaps be more explicit?

method 1:
```js
if (String(e).includes("@aws-sdk/crc64-nvme-crt")) {
  ...
  continue;
}
```

method 2:

```js
// throw site
const error = new Error("...");
error.missingOptionalDependency = true;
throw error;
```

```js
// catch site
if (error.missingOptionalDependency) {
  ...
  continue;
}
```